pkg/controller/serverside/reconcile.go

```diff
@@ -1,16 +1,18 @@
-package reconciler
+package serverside
```

Comment:
Are these controller reconcilers always server side operations? Do we want to make this distinction because these methods aren't always called and used in special cases?

Reply:
Sorry, I don't follow, what do you mean exactly by making a distinction?

In case it helps, this refers to server side apply provided by the kuber api server.